packages/icon-button-react/README.md

```diff
@@ -0,0 +1,49 @@
+# [`@fremtind/jkl-button-react`](https://fremtind.github.io/jokul/komponenter/buttons)
```

Comment:
Når det er egen pakke, bør den vel ha sin egen seksjon i portalen.

Men hva er grunnlaget for at denne er sin egen pakke isteden for å være en komponent i Buttons?